Stripe/PublicHeaders/STPApplePayContext.h

```diff
@@ -61,6 +66,8 @@ didSelectShippingMethod:(PKShippingMethod *)shippingMethod
 /**
  Called when the user has selected a new shipping address.  You should inspect the
  address and must invoke the completion block with an updated array of PKPaymentSummaryItem objects.
+ 
+ @note This does not contain full contact information - you can only receive that after the user authorizes payment, in the paymentInformation passed to `applePayContext:didCreatePaymentMethod:paymentInformation:completion:`
```

Comment:
the distinction here isn't really clear to me. Could we enumerate what is available vs what isn't available here in a little more detail?